apps/finevent/finevent.star

```diff
@@ -0,0 +1,580 @@
+    "Spain": "es",
+    "Sri Lanka": "lk",
+    "Sudan": "sd",
+    "Suriname": "sr",
+    "Svalbard and Jan Mayen": "sj",
+    "Swaziland": "sz",
+    "Sweden": "se",
+    "Switzerland": "ch",
+    "Syrian Arab Republic": "sy",
+    "Taiwan, Province of China": "tw",
+    "Taiwan": "tw",
+    "Tajikistan": "tj",
+    "Tanzania, United Republic of": "tz",
+    "Tanzania": "tz",
+    "Thailand": "th",
+    "Timor-Leste": "tl",
+    "Togo": "tg",
+    "Tokelau": "tk",
+    "Tonga": "to",
+    "Trinidad and Tobago": "tt",
+    "Tunisia": "tn",
+    "Turkey": "tr",
+    "Turkmenistan": "tm",
+    "Turks and Caicos Islands": "tc",
+    "Tuvalu": "tv",
+    "Uganda": "ug",
+    "Ukraine": "ua",
+    "United Arab Emirates": "ae",
+    "United Kingdom": "gb",
+    "United States": "us",
+    "United States Minor Outlying Islands": "um",
+    "Uruguay": "uy",
+    "Uzbekistan": "uz",
+    "Vanuatu": "vu",
+    "Venezuela, Bolivarian Republic of": "ve",
+    "Venezuela": "ve",
+    "Viet Nam": "vn",
+    "Vietnam": "vn",
+    "Virgin Islands, British": "vg",
+    "British Virgin Islands": "vg",
+    "Virgin Islands, U.S.": "vi",
+    "U.S. Virgin Islands": "vi",
+    "Wallis and Futuna": "wf",
+    "Western Sahara": "eh",
+    "Yemen": "ye",
+    "Zambia": "zm",
+    "Zimbabwe": "zw",
+}
+
+IMPORTANCE_ICONS = {
+    1: base64.decode("iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAAXNSR0IArs4c6QAAAEpJREFUKFNjZCASMOJTV1pa+h8k393dzYhXoZOT0395eXmG+fPnU6AQ2Tq8JiJLYij8n88AdjTjRAZG/ArDlf5/0/jKwN34ko4KAUofUQvpr/OmAAAAAElFTkSuQmCC"),
+    2: base64.decode("iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAAXNSR0IArs4c6QAAAFpJREFUKFNjZCASMOJTV1pa+h8k393dzYhXoZOT0395eXmG+fPnU6Dw/3I1sHWMkbcY8Zr4c4Xqf7YPtxkYMxgwFf7PZ4CYMpGBEb/CcKX/3zS+MnA3vqSjQgB3tVYLtGTHpgAAAABJRU5ErkJggg=="),
+    3: base64.decode("iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAAXNSR0IArs4c6QAAAFpJREFUKFNjZCASMOJT9/9j2X+QPCN/FyN+hT+P/Gf4c5OBkTuZAoX/l6tBrIu8xfgfn4k/V6j+Z/twm4ExgwFT4f98BogpExkY8SsMV/r/TeMrA3fjSzoqBABNA2YLBoHQ+QAAAABJRU5ErkJggg=="),
+}
+
+def flag_api(country_name):
+    cache_prefix = "%s/%s/%s/" % ("finevent", "econ", "flags")
+    flag = cache.get(cache_prefix + country_name)
+    if not flag:
+        print("Getting %s flag from the flag CDN, ISO3166 code: %s" % (country_name, ISO3166.get(country_name)))
+        flag_resp = http.get("https://flagcdn.com/w20/%s.png" % ISO3166.get(country_name))
+        if flag_resp.status_code != 200:
+            flag = ISO3166.get(country_name)
+        else:
+            flag = flag_resp.body()
+            cache.set(cache_prefix + country_name, flag, ttl_seconds = 60 * 60 * 24 * 30)  # keep for a month
+    return flag
+
+def random(max):
+    """Borrowed from nipterink's nft.star"""
+    return (time.now().nanosecond // 1000) % max
+
+def main(config):
+    timezone = config.get("$tz", "America/New_York")
+    countries = REGIONS.get(config.get("region"), [])
+    future_events = config.get("future") in (True, "true", "True")
```

Comment:
`config.bool()` will do the type conversion for you!

Reply:
Of course! Why didn't I think of that.... will change

Reply:
Fixed in 03afea9d6e643197a08c2d9b7e29458a85f1b82f